### DISCUSSION TITLE
Compiling Freetz FOS 8 for 7530

### DISCUSSION THREAD

**Discussion by gc-user** [Freetz-NG/freetz-ng#1054] [Allgemein]
As `make menuconfig` shows FOS 8 only for a few (3 to be precise) fritzboxes even though it has been released for 7530 as well, I checked around and found `/config/avm/availability.in` which seems to define that part of the content for `menugconfig` and  `/config/mod/dl-firmware.in` which seems to make sure the download during `make` works.

Is modifying these 2 files all that is required to successfully build a working FOS 8 freetz-ng image for 7530 - or is there more to it and I should rather wait than act the wannabe-pioneer? It builds fine, but I don't want to ruin my fritzbox either... so I rather ask before doing anything stupid.

Also, is there any significance to [AVM removing all source file archives for 7530](https://github.com/Freetz-NG/freetz-ng/commit/81b8bc2716b72af8c3331724dfb3220b79a2530d) which they haven't even done for devices that are EOS for years?

**Comment by fda77** [comment]
es fällt halt nicht vom himmel
